word_guess.rb

```diff
@@ -0,0 +1,163 @@
+		puts "  #{@move[@wrong_count]}  #{@g4} #{@spaces[@wrong_count]}  #{@p4}  ".colorize(:background => :black)
+		puts "===================================".colorize(:color => :blue, :background => :black)
+		puts "                                   ".colorize(:color => :yellow, :background => :black)
+		puts "Word: #{@game_word_array.join(" ")} #{@space1}".colorize(:color => :yellow, :background => :black)
+		puts "letters guessed: #{@guessed_letters.join(" ")} #{@space2}".colorize(:color => :yellow, :background => :black)
+		puts "Number of guesses left: #{5-@wrong_count}          ".colorize(:color => :yellow, :background => :black)
+		puts "===================================".colorize(:color => :blue, :background => :black)
+		puts
+	end
+
+
+	def guess
+
+		if @secret_word_array.include?(@letter)
+			@secret_word_array.each_index do |x|
+				if @secret_word_array[x] == @letter
+					@game_word_array[x] = @letter
+				end
+			end
+		else
+			@wrong_count += 1
+			@guessed_letters.push(@letter)
+		end
+	end
+
+	def game_over?
+		guess
+		if @wrong_count < 5
+			if @game_word_array.include?("_")
+				image
+				user_input
+			else
+				@space1 = " "*(27 - (@game_word_array.length*2))
+				@space2 = " "*(19 - (@guessed_letters.length*2))
+				@space3 = " "*(26 - (@game_word_array.length))
+				puts "===================================".colorize(:color => :green, :background => :black).blink
+				puts "   #{@move[@wrong_count]}  .-.  #{@spaces[@wrong_count]}  #{@p1}  ".colorize(:color => :green, :background => :black).blink
+				puts "   #{@move[@wrong_count]} | OO| #{@spaces[@wrong_count]} #{@p2} ".colorize(:color => :green, :background => :black).blink
+				puts %! #{@move[@wrong_count]}   |   | #{@dots[@wrong_count].colorize(:green)} #{@p3} !.colorize(:color => :green, :background => :black).blink
+				puts "  #{@move[@wrong_count]}  '^^^' #{@spaces[@wrong_count]}  #{@p4}  ".colorize(:color => :green, :background => :black).blink
+				puts "===================================".colorize(:color => :green, :background => :black).blink
+				puts "                                   ".colorize(:color => :green, :background => :black).blink
+				puts "Word: #{@game_word_array.join(" ")}   #{@space1}".colorize(:color => :green, :background => :black)
+				puts "letters guessed: #{@guessed_letters.join(" ")} #{@space2}".colorize(:color => :green, :background => :black)
+				puts "Word is #{@secret_word} #{@space3}".colorize(:color => :green, :background => :black)
+				puts "Number of guesses left: #{5-@wrong_count}          ".colorize(:color => :green, :background => :black) 
+				puts "===================================".colorize(:color => :green, :background => :black).blink 
+				puts "CONGRATS!!! YOU WON!!!!!!".colorize(:color => :green, :background => :black).blink
+				puts
+				exit
+			end
+
+		elsif @wrong_count == 5
```

Comment:
Can `@wrong_count` ever equal 6? If I'm following your program flow correctly, I don't think it can. Whether it can or not, I think a plain `else` is safer here than the `elsif` condition.

Because your game rule says that a person only gets five guesses, once `@wrong_count < 5` isn't true anymore, they should get a game over, regardless of whether `@wrong_count` is 6 or 60.